### PR TITLE
Feature keybind

### PR DESCRIPTION
**Pull request by joonalillfors**

### CODE REVIEW

File: client/src/components/comment-panel/comment-panel.js [hunk 1]
```diff
@@ -141,11 +142,12 @@ class CommentPanel extends React.Component {
         </div>
         <div className={css('panel-body')}>
           { this.commentContainer() }
-          <form className={css('comment-form')} onSubmit={this.handleSubmit}>
+          <form className={css('comment-form')} form="formie" onSubmit={this.handleSubmit}>
```
Comment (quvide) [inline client/src/components/comment-panel/comment-panel.js]: ???

File: client/src/globals.js [hunk 2]
```diff
@@ -31,3 +31,10 @@ export function getUsers() {
   return users
 }
 
+// Gets called onKeyDown from textArea of form element, if keys pressed are ctrl+enter or
+//  metaKey+enter, clicks parentelements input element which has attribute type="submit"
+export function keyPressSubmit(e) {
```
Comment (quvide) [inline client/src/globals.js:36]: Would it be better to give this event handler the submit button as an argument? It would be purer this way

**Review by quvide — CHANGES_REQUESTED** [review]

File: client/src/components/comment-panel/comment-panel.js (hunk 1 above)
Comment (joonalillfors) [reply client/src/components/comment-panel/comment-panel.js]: fixd

File: client/src/globals.js (hunk 2 above)
Comment (joonalillfors) [reply client/src/globals.js:36]: this way the place where this function is called needs less code and effort

Comment (quvide) [reply client/src/globals.js:36]: ok ok

**Review by quvide — APPROVED** [review]

**Comment by HenrikHelenius** [comment]
[[task] [misc] Enter + ctrl submits form Joona](https://trello.com/c/bL5VzwTW/149-task-misc-enter-ctrl-submits-form-joona)

**Pull request merged**